library/ix-dev/community/netbootxyz/values.yaml

```diff
@@ -0,0 +1,30 @@
+image:
+  repository: netbootxyz/netbootxyz
+  pullPolicy: IfNotPresent
+  tag: 0.7.0-nbxyz2
+
+resources:
+  limits:
+    cpu: 4000m
+    memory: 8Gi
+
+netbootConfig:
+  additionalEnvs: []
+
+netbootNetwork:
+  tftpPort: 30031
+  webHttpPort: 30032
+  webAssetsPort: 30033
```

Comment:
Use ports 31009+ (31008 is the last port used in my latest PR)
We try to have unique ports to avoid conflicts

Reply:
(Also update them on questions.yaml)